README.md

```diff
@@ -0,0 +1,86 @@
+# PyQtWebEngine BaseApp
+
+This base application is designed to be used for packaging a Flatpak application that depends on PyQt or PyQtWebEngine.
+
+**Note: This packaging is not affilated with, or supported by Riverbank Computing Limited.**
+
+## What's included?
+
+* [QtWebEngine base application](https://github.com/flathub/io.qt.qtwebengine.BaseApp)
+* [PyQt](https://riverbankcomputing.com/software/pyqt/) Python bindings for Qt
+* [PyQtWebEngine](https://riverbankcomputing.com/software/pyqtwebengine) Python bindings for QtWebEngine
+* Run-time depedencies for the above, including:
+  * freeglut
+  * glu
+  * krb5
+  * libevent
+  * python-opengl
+
+### Build tools and their dependencies
+
+Build tools are included to help package extra PytQt bindings and Python modules. These tools are removed by the  
+`BaseApp-cleanup.sh` script.
+
+#### Python module build tools
+
+* python-build
+* python-flit-core
+* python-installer
+* python-packaging
+* python-pep517
+* python-setuptools-scm
+
+#### PyQt build tools
+
+* pyqt-builder
+* pyqt-sip
+* sip
+
+#### Retained Python modules
+
+While the following Python modules are depedencies of the mentioned build tools, they are not removed by the  
+`BaseApp-cleanup.sh` script, as they might be needed by the Flatpak application.
+
+* python-pyparsing
+* python-tomli
+* python-toml
```

Comment:
We can build pyqt and pyqt webengine in another prefix, then install to `${FLATPAK_DEST}/lib/${PY_VER}/site-packages`  
So, we can remove all build dependencies with `cleanup: ["*"]`

Reply:
> We can build pyqt and pyqt webengine in another prefix, then install to ${FLATPAK_DEST}/lib/${PY_VER}/site-packages.

I'm not sure that I follow. We want to give application maintainers build/packaging tools for installing Python modules and extra PyQt bindings (e.g. PyQtDataVisualization).  
This means that Python modules like python-tomli will need to be distributed with the base application.  
Having these Python modules already in the base app, and then removed by the BaseApp-cleanup.sh` script will break applications that do need them, or enable features as these modules are available during make build.